packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -215,6 +217,8 @@ class Chat extends Widget<Properties> {
       case 'onMessageSend':
         this._createMessageSendAction();
         break;
+      case 'showDateHeaders':
```

Comment:
let's publish this option immediately.
So please, add runtime processing and tests for this.
For .d.ts - let's do in a separate PR